### ISSUE TITLE
triggerUntil() breaks ajax calls

### ISSUE THREAD

**Issue by Michal-sk** [zfcampus/zf-apigility#96]
Since `Zend\EventManager\EventManager::triggerUntil()` is deprecated and triggers and deprecated message, it breaks the ajax calls in the admin UI when you chose to display the deprecated messages.

It's only called in `ZF\Rest\Resource`

**Comment by codeaid** [comment]
You could set your `error_reporting` in your `php.ini` file to:

```
error_reporting = E_ALL & ~E_DEPRECATED
```

It would only act as a bandaid though and Apigility needs to be updated to not use the `triggerUntil()` method any more. It could already be fixed in 1.1, we'll see soon.

**Comment by Michal-sk** [comment]
Well dependens on how you like to work. I like having them all on an working in a manner that they don't show up.

But for those who would like that as a fix, i would than rather suggest:
```
error_reporting = E_ALL & ~ E_USER_DEPRECATED
```

**Comment by weierophinney** [comment]
All instances of these were fixed for the 1.1 version. If you are on a previous version, make sure you have `display_errors` disabled and/or modify your error reporting to mask `E_USER_DEPRECATED` errors.
